persistence/ravendb/cluster-configuration-info.include.md

```diff
@@ -0,0 +1 @@
+INFO: Using RavenDB version 4 and higher in a cluster configuration with multiple nodes is only supported from Version 7 or higher. For more information, read [cluster configuration with multiple nodes not supported](/persistence/ravendb/cluster-configuration.md).
```

Comment:
```suggestion
INFO: Using RavenDB version 5 and higher in a cluster configuration with multiple nodes is only supported from Version 7 or higher. For more information, read [cluster configuration with multiple nodes not supported](/persistence/ravendb/cluster-configuration.md).
```

Reply:
Good catch. My brain was like "yeah cluster is there since v4 so let's leave it at that. But then Version 7 or higher doesn't make sense anymore"